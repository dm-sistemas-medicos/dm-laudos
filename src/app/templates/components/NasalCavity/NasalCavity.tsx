import ExamItem from "../ExamItem/ExamItem";
import Comments from "../Comments/Comments";

import parentStyles from "../../layout.module.scss";

const NasalCavity = () => (
  <fieldset data-testid="nasal-cavity-container">
    <h2 className={parentStyles["exam-section-title"]}>Cavidade Nasal</h2>
    <ul className={parentStyles["exam-item-list"]}>
      <ExamItem
        id="nasal-cavity-nasal-septum"
        label="Septo nasal"
        initialText="com desvio para direita, com esporão ósseo associado, parcialmente obstrutivo"
        rows={2}
      />
      <ExamItem
        id="nasal-cavity-middle-turbinates"
        label="Cornetos médios"
        initialText="normotróficos"
      />
      <ExamItem
        id="nasal-cavity-inferior-turbinates"
        label="Cornetos inferiores"
        initialText="hipertróficos"
      />
      <ExamItem
        id="nasal-cavity-secretion"
        label="Secreção"
        initialText="aquosa"
      />
      <ExamItem id="nasal-cavity-mucosa" label="Mucosa" initialText="pálida" />
      <Comments parentId="nasal-cavity" />
    </ul>
  </fieldset>
);

export default NasalCavity;
