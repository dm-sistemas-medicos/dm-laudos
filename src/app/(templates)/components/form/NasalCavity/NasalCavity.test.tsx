import "@testing-library/jest-dom";

import { render } from "@testing-library/react";

import NasalCavity from "./NasalCavity";

describe("NasalCavity component", () => {
  it("renders the component", () => {
    const { container } = render(<NasalCavity />);

    expect(container).toMatchSnapshot();
  });
});
