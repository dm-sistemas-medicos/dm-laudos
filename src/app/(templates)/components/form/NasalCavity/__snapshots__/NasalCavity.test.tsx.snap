// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NasalCavity component renders the component 1`] = `
<div>
  <fieldset>
    <h2
      class="exam-section-title"
    >
      Cavidade Nasal
    </h2>
    <ul
      class="exam-item-list"
    >
      <li
        class="exam-item"
        data-testid="nasal-cavity-nasal-septum-container"
      >
        <label
          for="nasal-cavity-nasal-septum"
        >
          Septo nasal
        </label>
        <textarea
          data-testid="nasal-cavity-nasal-septum"
          id="nasal-cavity-nasal-septum"
          rows="2"
        >
          com desvio para direita, com esporão ósseo associado, parcialmente obstrutivo
        </textarea>
      </li>
      <li
        class="exam-item"
        data-testid="nasal-cavity-middle-turbinates-container"
      >
        <label
          for="nasal-cavity-middle-turbinates"
        >
          Cornetos médios
        </label>
        <textarea
          data-testid="nasal-cavity-middle-turbinates"
          id="nasal-cavity-middle-turbinates"
          rows="1"
        >
          normotróficos
        </textarea>
      </li>
      <li
        class="exam-item"
        data-testid="nasal-cavity-inferior-turbinates-container"
      >
        <label
          for="nasal-cavity-inferior-turbinates"
        >
          Cornetos inferiores
        </label>
        <textarea
          data-testid="nasal-cavity-inferior-turbinates"
          id="nasal-cavity-inferior-turbinates"
          rows="1"
        >
          hipertróficos
        </textarea>
      </li>
      <li
        class="exam-item"
        data-testid="nasal-cavity-secretion-container"
      >
        <label
          for="nasal-cavity-secretion"
        >
          Secreção
        </label>
        <textarea
          data-testid="nasal-cavity-secretion"
          id="nasal-cavity-secretion"
          rows="1"
        >
          aquosa
        </textarea>
      </li>
      <li
        class="exam-item"
        data-testid="nasal-cavity-mucosa-container"
      >
        <label
          for="nasal-cavity-mucosa"
        >
          Mucosa
        </label>
        <textarea
          data-testid="nasal-cavity-mucosa"
          id="nasal-cavity-mucosa"
          rows="1"
        >
          pálida
        </textarea>
      </li>
      <li
        class="exam-item non-printable"
        data-testid="nasal-cavity-comments-container"
      >
        <textarea
          class="exam-item-double"
          data-testid="nasal-cavity-comments"
          id="nasal-cavity-comments"
          placeholder="Observações"
          rows="1"
        />
      </li>
    </ul>
  </fieldset>
</div>
`;
